governance-charter.md

```diff
@@ -159,19 +164,18 @@ to the respective positions.
 
 ### Maximal representation
 
-To encourage diversity there will be a maximum of one-third representation on
-the Governance Committee from any one company at any time. If the outcomes of
-an election result in greater than 1/3 representation (or maximum of two,
-whichever is greater), the lowest vote getters from any particular company will
-be removed until representation on the committee is less than one-third.
-
-If percentages shift because of job changes, acquisitions, or other events,
-sufficient members of the committee must resign until max one-third
-representation is achieved. If it is impossible to find sufficient members to
-resign, the entire company’s representation will be removed and new special
-elections held. In the event of a question of company membership (for example
-evaluating independence of corporate subsidiaries) a majority of all
-non-involved Governance Committee members will decide. 
+To encourage diversity there will be a maximum of two Governance Committee
+members from any one company at any time. If the outcomes of an election result
+in more than two candidates from the same company being elected, the lowest vote
+getters from any particular company will be removed.
+
+If representation changes because of job changes, acquisitions, or other events,
+sufficient members of the committee must resign until the maximum representation
+is respected. If it is impossible to find sufficient members to resign, the
```

Comment:
```suggestion
If representation changes because of job changes, acquisitions, or other events,
sufficient members of the committee must resign within 28 days until the maximum representation
is respected. If it is impossible to find sufficient members to resign after that period, the
```

I suggest to introduce a fixed period (vs a "as soon as possible"), this (a) allows the affected GC members to make up their mind and coordinate who stays & who resigns, (b) gives a clear end-date for when the removal will be applied which removes ambiguity when it will be applied (immediatly? After the remaining GC says "now"?) and (c) it may make things easier if upcoming elections are close.

I suggest days as a unit of measure since "weeks" or "months" have debatable length.

For the length of the period I think it shouldn't be a lot less than 21 days, since individuals may be away for 2+ weeks (long vacation, medical situation, etc.) and it allows to form a special election comittee in preparation already. I also think more than 42 days (~6 weeks) shouldn't be applied, to get this undesired state resolved quickly.

Reply:
I like the suggestion, incorporating.